quic/transport/ngtcp2/native/certificateverifier/insecure.nim

```diff
@@ -0,0 +1,15 @@
+import ngtcp2
+import ./certificateverifier
+
+type InsecureCertificateVerifier* = ref object of CertificateVerifier
+
+proc init*(t: typedesc[InsecureCertificateVerifier]): InsecureCertificateVerifier {.gcsafe.} =
+  return InsecureCertificateVerifier()
+
+method destroy*(t: InsecureCertificateVerifier) {.gcsafe.} =
+  discard
+
+method getPtlsVerifyCertificateT*(
+    t: InsecureCertificateVerifier
+): ptr ptls_verify_certificate_t =
+  return nil
```

Comment:
would it make sense to return function (verifier) that always return `true`?

Reply:
The advantage of returning `nil` is that picotls will immediatly skip all this branch: https://github.com/h2o/picotls/blob/master/lib/picotls.c#L3279-L3291

Reply:
Done in https://github.com/vacp2p/nim-quic/pull/56/commits/c3e770959ed02a6d53cf978db4485f7dcce1a473